src/components/InquiryInput.jsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from "react";
+import "./InquiryInput.css";
+
+const InquiryInput = () => {
+  const [text, setText] = useState("");
+
+  return (
+    <div className="inquiry-input">
+      <h4 className="inquiry-input__title">문의하기</h4>
+      <textarea
+        className="inquiry-input__notice"
+        placeholder="개인정보를 공유 및 요청하거나, 명예 훼손, 무단 광고, 불법 정보 유포시 모니터링 후 삭제될 수 있으며, 이에 대한 민형사상 책임은 게시자에게 있습니다."
+        value={text}
+        onChange={(e) => setText(e.target.value)}
+      />
+      <div className="inquiry-input__form">
+        <button className={`inquiry-input__submit ${text.trim() ? "inquiry-input__submit--active" : ""}`}>등록</button>
```

Comment:
💊 제안
문의하기가 text가 없을때 비활성화가 될 수 있게 해주세요~
또한 동적 class를 활용하시기보다 css 가상클래스 :disabled를 통해 구현하시는 것이 더 적절할 것 같습니다.

```suggestion
        <button disabled={!!text.trim()} className='inquiry-input__submit'>등록</button>
```